docs/source/workflow_inputs.rst

```diff
@@ -1346,8 +1346,11 @@ Each window or glass door area is entered as a ``/HPXML/Building/BuildingDetails
   .. [#] Orientation choices are "northeast", "east", "southeast", "south", "southwest", "west", "northwest", or "north".
   .. [#] GlassLayers choices are "single-pane", "double-pane", "triple-pane", or "glass block".
   .. [#] Summer vs winter shading seasons are determined per :ref:`shadingcontrol`.
-  .. [#] InteriorShading/SummerShadingCoefficient default value indicates 30% reduction in solar heat gain, based on `ANSI/RESNET/ICC 301-2019 <https://codes.iccsafe.org/content/RESNET3012019P1>`_.
-  .. [#] InteriorShading/WinterShadingCoefficient default value indicates 15% reduction in solar heat gain, based on `ANSI/RESNET/ICC 301-2019 <https://codes.iccsafe.org/content/RESNET3012019P1>`_.
+  .. [#] InteriorShading/SummerShadingCoefficient default value is calculated with shgc, based on `ANSI/RESNET/ICC 301-2022 Addendum C <https://codes.iccsafe.org/content/RESNET3012022P1>`_: 
+  
```

Comment:
I didn't find a link for Addendum C so I just put the ANSI 301-2022 here.

Reply:
We've just been excluding any link for Addendum C since it's not official out yet. I'll remove the link and make a couple other minor changes.